packages/cli/generators/model/templates/model.ts.ejs

```diff
@@ -1,7 +1,12 @@
-import {Entity, model, property} from '@loopback/repository';
+<% if (isModelBaseBuiltin) { -%>
+import {<%= modelBaseClass %>, model, property} from '@loopback/repository';
```

Comment:
We should skip `property` import if no properties are defined for the model to avoid tslint errors.

Reply:
What if the developer creates an empty model and then he starts adding the properties manually in the generated class?